src/config/dynamic-theme-fixes.config

```diff
@@ -689,6 +689,13 @@ INVERT
 
 ================================
 
+account.booking.com
```

Comment:
The whole page of booking has this black logo problem as I checked.

I propose remove this part with account.booking com then:
- go to booking.com part of fixes
- add `*.booking.com` line under booking.com to include all subdomains (account will be included)
- add span[data-testid="header-logo"] into the INVERT section there.

I don't see need of separate subpage fix if it's needed for whole booking.com 😉

Reply:
Understood. Merged into the other :)